tests/functions/test_change_type.py

```diff
@@ -1,6 +1,12 @@
+<<<<<<< HEAD
```

Comment:
@dave-frazzetto this conflict needs to be resolved.

Can you delete these lines: 1, 2, 3, 4, 5, 9?